### PR TITLE
CB2-7793: Add approval region lookup table

### PR DESCRIPTION
**Pull request by Ian-Hodges**
## Description

Adds approval_region lookup table and populates it from seed data. Can be used with technical_record.approvalType to determine the region the trailer was approved in.

image

Related issue: [CB2-7793](https://dvsa.atlassian.net/browse/CB2-7793)

### CODE REVIEW

File: changelog-master.xml [hunk 1]
```diff
@@ -9,5 +9,7 @@
     <include  file="sql/00002_test_result_audit_test_type_version_trigger.sql" relativeToChangelogFile="true"/>
     <include  file="sql/00003_test_result_audit_test_type_version_trigger.sql" relativeToChangelogFile="true"/>
     <include  file="sql/00004_auth_into_service_table.sql" relativeToChangelogFile="true"/>
+    <include  file="sql/00005_approval_region_table.sql" relativeToChangelogFile="true"/>
     <include  file="sql/10000_views.sql" relativeToChangelogFile="true"/>
+    <include  file="seedData/seedData.xml" relativeToChangelogFile="true"/>
```
Comment (ChrisPeska-BJSS) [inline changelog-master.xml:14]: Really petty thing on naming note - if I had no context what it is for I would succumb to thinking it is some sort of Test Data - would it be more descriptive if we call it `reference_data` ?

**Review by ChrisPeska-BJSS — COMMENTED** [review]
👍

File: changelog-master.xml (hunk 1 above)
Comment (Ian-Hodges) [reply changelog-master.xml:14]: Very good point. It's too generic too. Doesn't leave any scope for new reference tables in the future.

Comment (Ian-Hodges) [reply changelog-master.xml]: Ah, it's not generic. Getting my files mixed up! Changed **seed** to **reference**.

**Review by gjulien-bjss — APPROVED** [review]

**Review by kcarr-bjss — APPROVED** [review]

**Pull request merged**